nginx-php-moodle/bin/watch_certificate_dir.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/with-contenv bash
+#
+# Copyright (c) 2018 SD Elements Inc.
+#
+#  All Rights Reserved.
+#
+# NOTICE:  All information contained herein is, and remains
+# the property of SD Elements Incorporated and its suppliers,
+# if any.  The intellectual and technical concepts contained
+# herein are proprietary to SD Elements Incorporated
+# and its suppliers and may be covered by U.S., Canadian and other Patents,
+# patents in process, and are protected by trade secret or copyright law.
+# Dissemination of this information or reproduction of this material
+# is strictly forbidden unless prior written permission is obtained
+# from SD Elements Inc..
+# Version
+
+# Source shtdlib
+# shellcheck disable=SC1091
+source /usr/local/bin/shtdlib.sh
+
+echo "Watching ${CERT_DIR} for updated certs..."
+
+function reload_nginx {
+    echo "Detected files modified in certificate directory..."
+    echo "Linking certificate files..."
+    ln -sf "${CERT_DIR}/${CERT_DOMAIN}.fullchain.pem" /etc/nginx/ssl/moodle.crt
+    ln -sf "${CERT_DIR}/${CERT_DOMAIN}.key.pem" /etc/nginx/ssl/moodle.key
+    echo "Reloading nginx..."
+    sleep 10
```

Comment:
Why the 10s sleep?

Reply:
The add_on_mod function sometimes acts to fast and seems to try and run the reload before the files are in place.

Reply:
Ah.. OK. Thats not ideal but if there is no way around it then move the sleep to the beginning of the function with a comment saying why its there.. E.g. `# Sleep for 10s until all files are updated`